lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5379,7 +5379,11 @@ static void ensureRequirementsAreSatisfied(ASTContext &ctx,
 
       auto outerIsolation = conformance->getIsolation();
       ProtocolConformanceRef(assocConf).forEachIsolatedConformance(
-          [&](ProtocolConformance *isolatedConformance) {
+          [&](ProtocolConformanceRef isolatedConformanceRef) {
+            if (!isolatedConformanceRef.isConcrete())
```

Comment:
Can this be a pack conformance?

Reply:
No, we look through them for this callback. I'll be back in this code shortly.